pyicloud/services/account.py

```diff
@@ -207,23 +192,17 @@ def __getitem__(self, key):
             return self._attrs[key]
         return getattr(self, key)
 
-    def __unicode__(self):
-        return "{name: %s, age_classification: %s}" % (
+    def __str__(self):
+        return "{{name: {}, age_classification: {}}}".format(
```

Comment:
f-string

Reply:
I'm tempted to change the format of all these to `"<name:...>"`, to match the usual python style (also used in most other classes too). Thoughs?